DEPLOYMENT.md

```diff
@@ -0,0 +1,66 @@
+# Configuring RHMAP Core to use non-wildcard SSL routing
+
+For a standard RHMAP installation deployed on top of OpenShift, we recommend that a wild card SSL certificate be installed on the OpenShift router. However, in situations where this is not desirable, RHMAP can be exposed using a single URL.
+
+## RHMAP Proxy
+
+RHMAP Proxy is a separate component which should be deployed to OpenShift per MBaaS project. Its purpose is to route requests from a single external host to cloud applications and RHMAP Core instances deployed on OpenShift.
+
+
+For example, when RHMAP Proxy receives a request for `https://rhmapproxy.internal.com/cloud_app_id/hello`, it will be translated and reverse proxied to `http://cloud_app_id.internal.com/hello`.
+
+The RHMAP Proxy application is bundled within the RHMAP RPM in the form of an OpenShift template. Creating this application will create an OpenShift route which will expose this service. Carry out the following steps to configure RHMAP Core to use non-wildcard routing.
+
+**_1.  Set the EXTERNAL_HOST environment variable for client side connectivity_**
+
+Upon initialization, a client application must call out to RHMAP Core to discover the URL to which it should make API requests. We must set the following environment variable in the `millicore` component to allow this to work behind RHMAP Proxy.
+
+```
+oc env dc millicore -n rhmap-core EXTERNAL_HOST=https://rhmaprouter.internal.domain.com
+```
+
+NOTE: The `EXTERNAL_HOST` value must match the value of the route which you will pass to the template for RHMAP Proxy
```

Comment:
The `EXTERNAL_HOST` environment variable represents url that would be exposed to internet. For example... 
URL will be the main entry point for all platform applications and it would be used to retrieve API url on application startup.